### ISSUE TITLE
[CAIA A11y Research - Pilot] MHV on VA.GOV, Medical Records (March 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#78209]
> [!IMPORTANT]
> - **TICKET STATUS:** `Rescheduled`
> - **PILOT DATES:**  ~March 15, 2024 11 a.m and 1 p.m.~
>     - **Rescheduled Dates**: 
>         - Thursday, March 28, 3:30- 5 p.m. 
>         - Friday, March 29, 1 - 2:30 p.m and/or 3-4:30 p.m.
> 
> - **TYPES OF ASSISTIVE TECH:** Actual Users of Screen Readers one will be Mobile `iOS VoiceOver` - (Pilot 1) and the other will be Desktop `JAWS` (Pilot 2)

# CAIA A11y Support
This is a child ticket of the research ticket [77076](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77076) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.
> - This pilot is unique, in that we have two native users of screen readers participating in the pilot. 
> - There will be two pilots as well. 

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @EvanAtCoforma 
- @SarahKay8 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: MHV on VA.GOV
- OCTO-DE product owner: Kaitlin Fink
- Product manager:Marci McGuire
- Designer: Lexi Wunder
- FE engineer:
- Product/team Slack channel: [mhv-medical-records](https://dsva.slack.com/archives/C03Q2UQL1AS)
- MHV a11y: Bobby Bailey
- Accessibility Slack Channel for MHV: [mhv-accessibility](https://dsva.slack.com/archives/C05QG2H510T)

</details>

## Description

> [!CAUTION]
> ### Heads Up: Pilot Awareness Items
> - **New information to keep in mind:**
>     - Allergies will not be included due to staging issues. 
>     - The conversation guide will be updated to reflect the removal of allergies, before the AT sessions. 
>     - The records in staging may not match the records in the prototype. (It might not be as long, or the content may not be correct.) However, the tasks will be similar. Discrepancy won't impact the task, so this was deemed ok to proceed, so just a heads up on that. 

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- ~**Pilot 1:** March 15, 2024 at 11 a.m. Mobile (Talkback)~ `Rescheduled` 
- ~**Pilot 2:** March 15, 2024 at 1 p.m. Desktop~ `Rescheduled` 
- **Pilot 1:** Thursday, March 28, 3:30- 5 p.m. 
- **Pilot 2:** Friday, March 29, 1 - 2:30 p.m and/or 3-4:30 p.m.
- **Prototype**: [Staging Link](https://staging.va.gov/my-health/medical-records/)
- **Research Convo Guide**: [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-03-usability-testing-rd2-at/conversation-guide.md)
- **Research Plan**: [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-03-usability-testing-rd2-at/research-plan.md)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11ys:** Evan and SK
- [x] **Determine date and time for pilot:** March 15, 2024
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [ ] **Review research** [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-03-usability-testing-rd2-at/research-plan.md)
- [ ] **Review research** [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-03-usability-testing-rd2-at/conversation-guide.md) and provide feedback in comments.
- [ ] [Requested AT device and tech to be used in sessions via main ticket 77076](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77076)
- [x] [Receive AT device and tech from Team in main research ticket 77076](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77076) to be used in sessions to prep for pilot `iOS VoiceOver` and `JAWS`
- [ ] **Review prototype** in `Staging` -  [Staging Link](https://staging.va.gov/my-health/medical-records/)
- [ ] **Run pilot 1** with VFS Researcher
- [ ] **Run pilot 2** with VFS Researcher
- [ ] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. Folder for Team in GH Deliverables for CAIA A11Ys.
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/78985
```

**Comment by sara-amanda** [comment]
## Rescheduled Pilots
- This ticket for the **Medical Records Pilot Sessions** has been updated to reflect the rescheduling of the pilot sessions from 3/15 to 3/28 and 3/29, per @AdeAjayii [comment in parent ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77076#issuecomment-2018036488).
- The **CAIA Capacity Calendar** has been updated with three holds, pending official invites. 

cc: @SarahKay8 @EvanAtCoforma @coforma-terry

**Comment by sara-amanda** [comment]
Closing ticket
